host.json

```diff
@@ -29,11 +29,11 @@
   "logging": {
     "fileLoggingMode": "debugOnly",
     "logLevel": {
-      "default": "None",
       "Host.Results": "Error",
-      "Function.ProcessFDR1BlobFiles": "Information",
-      "Function.ProcessFDR3BlobFiles": "Information",
-      "Function.HTTPBlobRecovery": "Information",
+      "Function.ProcessFDR1BlobFiles": "Error",
```

Comment:
Why not using: 
`AzureFunctionsJobHost__logging__logLevel__Function__ProcessFDR1BlobFiles: "<VALUE>"`
in Helm environment file and make it more customizable? If values in Helm file are changed for debug purposes, they override that value IIRC. 
Ditto for the other functions.